components/sensor/sgp40.rst

```diff
@@ -0,0 +1,57 @@
+SGP40 Volatile Organic Compound Sensor
+==============================================
+
+.. seo::
+    :description: Instructions for setting up SGP40 Volatile Organic Compound sensor
+    :image: sgp40.jpg
+
+The ``sgp40`` sensor platform  allows you to use your Sensiron SGP40 VOC sensor
+(`datasheet <https://www.sensirion.com/fileadmin/user_upload/customers/sensirion/Dokumente/9_Gas_Sensors/Sensirion_Gas_Sensors_SGP40_Datasheet.pdf>`__) with ESPHome.
+The :ref:`I²C Bus <i2c>` is required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/sgp40.jpg
+    :align: center
+    :width: 80.0%
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sensor:
+      - platform: sgp40
+        name: "Workshop VOC"
+        update_interval: 5s
```

Comment:
The first example should be a minimal configuration, remove optional parameters unless necessary for a working configuration.

You can add a section at the end specifying a sample configuration using optional parameters to show off component features.

Reply:
Ok